sensor_msgs/PointCloud2 cloud
---
lvr_ros/PolygonMesh polymesh
